apps/configurator/lib/configurator_web/controllers/skill_html/edit.html.heex

```diff
@@ -0,0 +1,8 @@
+<.header>
+  Edit Skill <%= @skill.name %>
+  <:subtitle>Use this form to manage skill records in your database.</:subtitle>
```

Comment:
Are this subtitle really necessary?

Reply:
What do you mean? The text or the tag?

Reply:
The whole tag, i just noticed that this subtitle is auto-generated so we can just leave it